package.json

```diff
@@ -38,6 +38,7 @@
   "dependencies": {
     "@sentry/node": "^5.6.2",
     "@spectacles/rest": "^0.5.2",
+    "axios": "^0.19.0",
```

Comment:
we already use `node-fetch`, on line 51

Reply:
Ok, I can refactor to use that one instead